### PR TITLE
submit blog entry describing `bixu/cacher` package

### PR DESCRIPTION
**Pull request by bixu**
Submitting at @eeyun's request :D

Signed-off-by: Blake Irvin <blake.irvin@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/blog/2018-08-30-Introducing-Cacher.html.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+With cacher, second run:
+  api: Build time: 6m56s
+
+With cacher, third run:
+  api: Build time: 6m4s
+```
+**Build performance improves in this case by ~25%. ❤**
+
+Right now, `cacher` only the depdency managers discussed here, but any dependency manager who's behavior can be configured using environment variables could be  supported in future. Pull requests are most welcome!
```
Comment (christophermaier) [inline www/source/blog/2018-08-30-Introducing-Cacher.html.md]: "Right now, `cacher` only the depdency managers" => "Right now, `cacher` only supports the dependency managers" 

(add "supports", and fix typo for "dependency")

"who's" => "whose"

File: www/source/blog/2018-08-30-Introducing-Cacher.html.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+With cacher, first run:
+  api: Build time: 9m31s
+
+With cacher, second run:
+  api: Build time: 6m56s
+
+With cacher, third run:
+  api: Build time: 6m4s
+```
+**Build performance improves in this case by ~25%. ❤**
```
Comment (christophermaier) [inline www/source/blog/2018-08-30-Introducing-Cacher.html.md]: Was the heart glyph intentional?

File: www/source/blog/2018-08-30-Introducing-Cacher.html.md (hunk 1 above)
Comment (bixu) [reply www/source/blog/2018-08-30-Introducing-Cacher.html.md]: Good catch!

File: www/source/blog/2018-08-30-Introducing-Cacher.html.md (hunk 2 above)
Comment (bixu) [reply www/source/blog/2018-08-30-Introducing-Cacher.html.md]: Yes, but it doesn't render well, so I'm pulling it out.

**Comment by bixu** [comment]
@christophermaier, we should be good to merge now I think.

**Comment by christophermaier** [comment]
Thanks @bixu!

**Pull request merged**